docs/_templates/overrides/metpy.calc.rst

```diff
@@ -0,0 +1,189 @@
+calc
+==========
+
+.. automodule:: metpy.calc
+
+
+   Dry Thermodynamics
+   ------------------
+
+   .. autosummary::
+      :toctree: ./
+
+      add_height_to_pressure
+      add_pressure_to_height
+      density
+      dry_lapse
+      dry_static_energy
+      geopotential_to_height
+      height_to_geopotential
+      height_to_pressure_std
+      mean_pressure_weighted
+      potential_temperature
+      pressure_to_height_std
+      sigma_to_pressure
+      static_stability
+      temperature_from_potential_temperature
+      thickness_hydrostatic
+
+
+   Moist Thermodynamics
+   --------------------
+
+   .. autosummary::
+      :toctree: ./
+
+      dewpoint
+      dewpoint_from_specific_humidity
+      dewpoint_rh
+      equivalent_potential_temperature
+      mixing_ratio
+      mixing_ratio_from_relative_humidity
+      mixing_ratio_from_specific_humidity
+      moist_lapse
+      moist_static_energy
+      precipitable_water
+      psychrometric_vapor_pressure_wet
+      relative_humidity_from_dewpoint
+      relative_humidity_from_mixing_ratio
+      relative_humidity_from_specific_humidity
+      relative_humidity_wet_psychrometric
+      saturation_equivalent_potential_temperature
+      saturation_mixing_ratio
+      saturation_vapor_pressure
+      specific_humidity_from_mixing_ratio
+      thickness_hydrostatic_from_relative_humidity
+      vapor_pressure
+      vertical_velocity
+      vertical_velocity_pressure
+      virtual_potential_temperature
+      virtual_temperature
+      wet_bulb_temperature
+
+
+   Soundings
```

Comment:
I was trying to label these "indicies" or similar since they could be used with model output, etc, but I'm not sure that's any clearer?

Reply:
I'm not really sure either...although I have frequently used "model soundings," and `parcel_profile` and `mixed_layer` don't really strike me as indices, so I'd probably still lean towards soundings...

Thoughts @kgoebber?